drivers/ethernet/eth_w5500.c

```diff
@@ -0,0 +1,586 @@
+/* W5500 Stand-alone Ethernet Controller with SPI
+ *
+ * Copyright (c) 2020 Linumiz
+ * Author: Parthiban Nallathambi <parthiban@linumiz.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT	wiznet_w5500
+
+#define LOG_MODULE_NAME	eth_w5500
+#define LOG_LEVEL	CONFIG_ETHERNET_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
Unless you really need LOG_MODULE_NAME or LOG_LEVEL variables in this driver, I suggest you write this like this:
`LOG_MODULE_REGISTER(eth_w5500, CONFIG_ETHERNET_LOG_LEVEL);`
and then you can remove the LOG_MODULE_NAME and LOG_LEVEL variables.

Reply:
Done thanks